<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dairy</class>
 <widget class="QDialog" name="Dairy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_searchdairy">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>551</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Searchdairy">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>80</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adddairyproduct">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_viewdairyproduct">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>690</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>ViewCart</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductname1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductprice1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductdesc1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductimagelink1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adddairyproduct_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductname2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductprice2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductdesc2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductimagelink2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adddairyproduct_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductname3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductprice3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductdesc3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductimagelink3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adddairyproduct_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>200</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductname4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductprice4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductdesc4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>350</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductimagelink4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>390</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adddairyproduct_5">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>200</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductname5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>260</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductprice5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>300</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductdesc5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>350</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductimagelink5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>390</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adddairyproduct_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductname6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductprice6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>540</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductdesc6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>580</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductimagelink6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>620</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adddairyproduct_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductname7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>510</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductprice7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>540</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductdesc7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>580</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductimagelink7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>620</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adddairyproduct_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductname8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductprice8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>540</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductdesc8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>580</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductimagelink8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>620</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adddairyproduct_9">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductname9">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>510</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductprice9">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>540</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductdesc9">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>580</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductimagelink9">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>620</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adddairyproduct_10">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>460</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductname10">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>510</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductprice10">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>540</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductdesc10">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>580</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dairyproductimagelink10">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>610</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_errormessagedairy">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff2600;&quot;&gt;The item you are searching for is not available or is currently not sold by any of our sellers and contributors&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
